src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,42 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("v1/products")
 public class ProductController {
+    private final ProductService productService;
+
+    public ProductService(ProductService productService) {
```

Comment:
co to jest?

Reply:
To ma być serwis produktów. W tej klasie chciałem zaimplementować metody wykorzystywane w kontrolerze